Sources/ArgumentParser/Parsing/ArgumentSet.swift

```diff
@@ -104,6 +104,14 @@ extension ArgumentSet {
     return ArgumentSet(arg)
   }
 
+  static func flag(key: InputKey, name: NameSpecification, help: ArgumentHelp?, value: Void) -> ArgumentSet {
```

Comment:
I'd also like to note that the `value: Void` argument is temporary, as the conflicting `Bool`-related flag method changes signature in #170, and this function will be `flag(key:name:help:)` after that.